### PR TITLE
chore(di): include globals in snapshots

### PR DESCRIPTION
**Pull request by P403n1x87**
We include the value of referenced globals in enriched log messages. We use the `statics` fields for carrying this extra information.

## Checklist

- [x] Change(s) are motivated and described in the PR description
- [x] Testing strategy is described if automated tests are not included in the PR
- [x] Risks are described (performance impact, potential for breakage, maintainability)
- [x] Change is maintainable (easy to change, telemetry, documentation)
- [x] [Library release note guidelines](https://ddtrace.readthedocs.io/en/stable/releasenotes.html) are followed or label `changelog/no-changelog` is set
- [x] Documentation is included (in-code, generated user docs, [public corp docs](https://github.com/DataDog/documentation/))
- [x] Backport labels are set (if [applicable](https://ddtrace.readthedocs.io/en/latest/contributing.html#backporting))
- [x] If this PR changes the public interface, I've notified `@DataDog/apm-tees`.

## Reviewer Checklist

- [x] Title is accurate
- [x] All changes are related to the pull request's stated goal
- [x] Description motivates each change
- [x] Avoids breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes
- [x] Testing strategy adequately addresses listed risks
- [x] Change is maintainable (easy to change, telemetry, documentation)
- [x] Release note makes sense to a user of the library
- [x] Author has acknowledged and discussed the performance implications of this PR as reported in the benchmarks PR comment
- [x] Backport labels are set in a manner that is consistent with the [release branch maintenance policy](https://ddtrace.readthedocs.io/en/latest/contributing.html#backporting)

### CODE REVIEW

**Review by shatzi — APPROVED** [review]
maybe we should align and check that those shows in the UI

File: ddtrace/debugging/_signal/snapshot.py [hunk 1]
```diff
@@ -50,18 +51,29 @@ def timeout(_):
...
-            if arguments is not None
+            if arguments
             else {},
             "locals": utils.capture_pairs(
                 _locals, limits.max_level, limits.max_len, limits.max_size, limits.max_fields, timeout
             )
-            if _locals is not None
+            if _locals
+            else {},
+            "statics": utils.capture_pairs(
```
Comment (P403n1x87) [inline ddtrace/debugging/_signal/snapshot.py:61]:
```suggestion
            "staticFields": utils.capture_pairs(
```

File: tests/debugging/test_debugger.py [hunk 2]
```diff
@@ -1197,6 +1197,7 @@ def test_debugger_redacted_identifiers():
                     "size": 3,
                 },
             },
+            "statics": {"SensitiveData": {"type": "type", "value": "<class 'tests.submod.stuff.SensitiveData'>"}},
```
Comment (P403n1x87) [inline tests/debugging/test_debugger.py:1200]:
```suggestion
            "staticFields": {"SensitiveData": {"type": "type", "value": "<class 'tests.submod.stuff.SensitiveData'>"}},
```

File: tests/debugging/test_encoding.py [hunk 3]
```diff
@@ -169,11 +169,12 @@ def test_capture_context_exc():
     try:
         raise Exception("test", "me")
     except Exception:
-        context = _capture_context([], [], sys.exc_info())
+        context = _capture_context([], [], [], sys.exc_info())
         exc = context.pop("throwable")
         assert context == {
             "arguments": {},
             "locals": {},
+            "statics": {},
```
Comment (P403n1x87) [inline tests/debugging/test_encoding.py:177]:
```suggestion
            "staticFields": {},
```

**Pull request merged**